src/tasks/tasks.service.spec.ts

```diff
@@ -15,4 +25,32 @@ describe('TasksService', () => {
   it('should be defined', () => {
     expect(service).toBeDefined();
   });
+
+  it('should call createTask method', () => {
+    const dto = new CreateTaskDto();
+    const createTaskSpy = jest.spyOn(service, 'createTask');
+    service.createTask(dto);
+    expect(createTaskSpy).toHaveBeenCalledWith(dto);
+  });
+
+  it('should call getTasks method with expected param', async () => {
+    const getTasksSpy = jest.spyOn(service, 'getTasks');
+    service.getTasks();
+    expect(getTasksSpy).toHaveBeenCalledWith();
+  });
+
+  it('should call findOneTask method with expected param', async () => {
```

Comment:
remove async if you are not using await or promise.then() or catch()

Reply:
Done